docs/configuration.rst

```diff
@@ -159,18 +159,25 @@ To create it manually, follow the specific deployment method documentation.
 
 To have the account created automatically, you just need to define a few environment variables:
 
-.. code-block:: bash
+- ``INITIAL_ADMIN_ACCOUNT``: the admin username: The first part of the e-mail address before the @.
+- ``INITIAL_ADMIN_DOMAIN``: the domain appendix: Most probably identical to the ``DOMAIN`` variable.
+- ``INITIAL_ADMIN_PW``: the admin password.
+- ``INITIAL_ADMIN_MODE``: use one of the options below for configuring how the admin account must be created:
+  
+  - ``create``: (default) creates a new admin account and raises an exception when already exists.
```

Comment:
(default) creates a new admin account and raises an exception when `it` already exists.

Reply:
Fixed. Thanks!